File: 준영/약수의 개수와 덧셈.js
function solution(left, right) {
  var answer = 0
  for (let i = left; i <= right; i++) {
    var idx = 1
    var cnt = 0
    while (idx <= i) {
      if (i % idx === 0) {
        cnt += 1
      }
      idx += 1
    }

    if (cnt % 2 === 0) {
      answer += i
    } else {
      answer -= i
    }
  }
  
  return answer
}

File: 준영/프로그래머스/LV1/약수의 개수와 덧셈.py
def div(n):
    idx = 1
    cnt = 0
    while idx <= n:
        if n % idx == 0:
            cnt += 1
        idx += 1
    return cnt
        

def solution(left, right):
    answer = 0
    for i in range(left, right+1):
        if div(i) % 2:
            answer -= i
        else:
            answer += i
            
    return answer